com/yadro/HWManager/StorageManager.interface.yaml

```diff
@@ -0,0 +1,24 @@
+description: >
+    The interface to control Yadro storage subsystem.
+
+methods:
```

Comment:
As we discussed earlier during our call, currently we're unable query the Disk Locator LED state.
As an alternative until if or when the MCU command is implemented, maybe we could modify the xyz....Item.Drive interface to store the LED status. As I understand, all disk-related data from that interface is stored in the storage.csv file and the LED status won't be lost should the service restart.

Reply:
The read ability would be implemented in next MCU firmware release and then the support would be added to the manager. As for now I can add "dummy" interface for this which you can use now.

I don't like the idea to remember state on BMC side since we not the one who control Location LEDs, they can also enabled from RAID control utility.
And, BTW, as for now I don't combine information from storage.csv and from MCU, they are fully independent yet.

Reply:
I agree with you on that. Just wondered whether we should have a temporary workaround. For now, I will just return a command-specific fail code on the query request, as mentioned in my task description.

Reply:
Added API function for this, see updated PR.